kratos/includes/kratos_parameters.h

```diff
@@ -815,7 +815,15 @@ class KRATOS_API(KRATOS_CORE) Parameters
     void ValidateAndAssignDefaults(const Parameters& rDefaultParameters);
```

Comment:
please revert the changes in the Parameters, those should no longer be necessary